src/runtime/composables/local/useAuth.ts

```diff
@@ -105,15 +105,19 @@ const getSession: GetSessionFunc<SessionData | null | void> = async (getSessionO
   return data.value
 }
 
-const signUp = async (credentials: Credentials, signInOptions?: SecondarySignInOptions) => {
+const signUp = async (credentials: Credentials, signInOptions?: SecondarySignInOptions, registerOptions?: RegisterOptions) => {
   const nuxt = useNuxtApp()
 
   const { path, method } = useTypedBackendConfig(useRuntimeConfig(), 'local').endpoints.signUp
-  await _fetch(nuxt, path, {
+  await _fetch<Record<string, any>>(nuxt, path, {
```

Comment:
Why do we add types to `_fetch` here? Generally speaking I am not a fan of adding any type with `any` as it is pretty ambiguous.

Reply:
Yeah, I am not fan too.
It's a refuse coming from the previous changes, I've added just like in the n.19 line.
I'm going to delete it 🚀